avocado/plugins/spawners/podman.py

```diff
@@ -94,7 +94,7 @@ def run(self, config):
         pass
 
 
-class PodmanSpawner(DeploymentSpawner, SpawnerMixin):
+class PodmanSpawner(Spawner, SpawnerMixin):
```

Comment:
I'm sure the move to the mount is a good move, but I have a mix feelings about making PodmanSpawer as a "non Deployment Spawner" anymore. I see this as "conceptual change" as well. I mean, the simple copy to a folders is kind of a "basic deployment", IMO.

So, my suggestion is try to keep using the "DeploymentSpawner" API here, as much as possible. If not possible at all, we should be clear about what is a DeploymentSpawner in our docstrings and docs to avoid confusion for other developers.